frontend/src/common/top_bar.tsx

```diff
@@ -21,13 +21,21 @@ export function MeButton() {
 export function MeMenu() {
   const { data, mutate } = useGetMe({});
   const [_location, navigate] = useLocation();
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
 
   return (
     <>
       <div className="pb-4">hello, {data?.username}</div>
       <PrimaryButton
         onClick={() => {
           storeAuthToken(undefined);
+
+          try {
+            const response = await logout({});
+          } catch (e) {
+            let message = 'An unknown error occcured.';
+            setErrorMessage(message);
+          }
```

Comment:
Since we don't use the error message, I'd be fine with just discarding it. What do you say @phlmn?